v2/azure-arm.yaml

```diff
@@ -1164,6 +1164,9 @@ objectModelConfiguration:
       PrivateDnsZone:
         $export: true
         $supportedFrom: v2.0.0-beta.2
+      PrivateDnsZones_VirtualNetworkLink:
```

Comment:
Do we also need to support DNS RecordSets? This is the URL in their Swagger:
`"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/privateDnsZones/{privateZoneName}/{recordType}/{relativeRecordSetName}":`

It's a bit funky because actually it's defining like 8 different resourceTypes there (`recordType` is an enum with a bunch of options including "A", etc).

I don't know what our generator will do with that, but do we need these resources too for users to create actual DNS entires?